week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,13 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var total = numberOfStudents + numberOfMentors;
+
+var studentPerc = (numberOfStudents * 100) / total;
+var mentorsPerc = (numberOfMentors * 100) / total;
+
+var roundStudentPerc = Math.round(studentPerc);
+var roundMentorsPecr = Math.round(mentorsPerc);
```

Comment:
Typo here :)

Great use of variables! Makes it really clear what the code is doing 👍 
Just a minot point about the naming - abbreviations are ok, but I feel writing out whole words is preferred - even though they might be long. That avoids any potential misinterpretations - clarity is key.